docs/data/date-pickers/digital-clock/digital-clock.md

```diff
@@ -0,0 +1,80 @@
+---
+product: date-pickers
+title: React Digital Clock component
+components: DigitalClock, MultiSectionDigitalClock
+githubLabel: 'component: TimePicker'
+packageName: '@mui/x-date-pickers'
+---
+
+# Digital Clock
+
+<p class="description">The Digital Clock components let the user select a time without any input or popper / modal.</p>
+
+## Description
+
+There are two component versions for different cases. The `DigitalClock` handles selection of a single time instance in one step, just like a `select` component. The `MultiSectionDigitalClock` allows selecting time using separate sections for separate views.
+
+The `DigitalClock` is more appropriate when there is a limited amount of time options needed, while the `MultiSectionDigitalClock` is suited for cases when a more granular time selection is needed.
+
+## Basic usage
+
+{{"demo": "DigitalClockBasic.js"}}
+
+## Uncontrolled vs. Controlled
+
+The components can be uncontrolled or controlled.
+
+{{"demo": "DigitalClockValue.js"}}
+
+## Form props
+
+The components can be disabled or read-only.
+
+{{"demo": "DigitalClockFormProps.js"}}
+
+## Views
+
+The `MultiSectionDigitalClock` component can contain three views: `hours`, `minutes`, and `seconds`.
+By default, only the `hours` and `minutes` views are enabled.
+
+You can customize the enabled views using the `views` prop.
+Views will appear in the order they're included in the `views` array.
+
+The `DigitalClock` component is built to work only with `hours` view, providing it with any more views will have no effect as the prop value will be ignored.
```

Comment:
Is it necessary? The `DigitalClock` has a single select already containing hours and minutes, so I would not expect to pass this `views` prop.

```suggestion
```

Reply:
Thank you for pointing it out. I think I'll simply extract only the `hours` view into the `DigitalClock` `TView` thus having two fronts to inform the user of it—prop types and the TS types.
What do you think? 🤔

Reply:
sounds good